Lib/ast.py

```diff
@@ -178,6 +186,7 @@ def _format(node, level=0):
         raise TypeError('expected AST, got %r' % node.__class__.__name__)
     if indent is not None and not isinstance(indent, str):
         indent = ' ' * indent
+    empty_values = {None, "''", "[]"}
```

Comment:
I don't think we should omit empty strings, since we don't use them as defaults:

```
>>> n=ast.Name(ctx=ast.Load())
<stdin>-1:1: DeprecationWarning: Name.__init__ missing 1 required positional argument: 'id'. This will become an error in Python 3.15.
>>> n.id
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
    n.id
AttributeError: 'Name' object has no attribute 'id'
```